Common/TableProducer/trackselection.cxx

```diff
@@ -65,6 +72,6 @@ struct TrackSelectionTask {
 //****************************************************************************************
 WorkflowSpec defineDataProcessing(ConfigContext const& cfgc)
 {
-  WorkflowSpec workflow{adaptAnalysisTask<TrackSelectionTask>(cfgc, TaskName{"track-selection"})};
+  WorkflowSpec workflow{adaptAnalysisTask<TrackSelectionTask>(cfgc)};
```

Comment:
Why?

Reply:
I thought that it was advised to remove `TaskName`, reverting this

Reply:
But then the name changes from track-selection to track-selection-task. Thanks for reverting.